prefect_aws/workers/glue_job_worker.py

```diff
@@ -0,0 +1,156 @@
+import logging
+import time
+from typing import Any, Optional
+
+import anyio
+from prefect.server.schemas.core import FlowRun
+from prefect.utilities.asyncutils import run_sync_in_worker_thread
+from prefect.workers.base import (
+    BaseJobConfiguration,
+    BaseVariables,
+    BaseWorker,
+    BaseWorkerResult,
+)
+from pydantic import Field
+
+from prefect_aws import AwsCredentials
+
+_GlueJobClient = Any
+
+
+class GlueJobWorkerConfiguration(BaseJobConfiguration):
+    """
+    Job configuration for a Glue Job.
+    """
+
+    job_name: str = Field(
+        ...,
+        title="AWS Glue Job Name",
+        description="The name of the job definition to use.",
+    )
```

Comment:
The `BaseJobConfiguration` class has a `name` attribute that automatically gets populated with the flow name. I think it'd be worthwhile to use that attribute rather than introducing another name attribute.
```suggestion 
```

Reply:
`job_name` is necessarily to use start glue job.
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/glue/client/start_job_run.html

Reply:
I am user very interested in using this feature, I actually I have forked this into my repo already. I think it's expected that the glue job exists already.

Reply:
@desertaxle 
Is the following direction correct?
1. delete Worker
2. modify the part that inherits Infrastructure to match the Interface of JobBlock

Reply:
Apologies for the delay. Yes, that is the direction that I'm suggesting.

Reply:
@desertaxle 
I have fixed it. However, the test is failing due to the boto3. I have not been able to reproduce this error in my local environment.